package.json

```diff
@@ -344,15 +344,17 @@
     "@brave/react-virtualized-auto-sizer": "^1.0.4",
     "@glif/filecoin-wallet-provider": "2.0.0-beta.12",
     "@ledgerhq/hw-app-eth": "6.24.1",
+    "@ledgerhq/hw-app-solana": "^6.27.1",
     "@ledgerhq/hw-transport-webhid": "6.24.1",
+    "@solana/web3.js": "1.36.0",
     "@types/jszip": "^3.1.6",
     "@types/parse-torrent": "^5.8.3",
     "@types/react-router-dom": "^5.1.9",
     "@types/webtorrent": "^0.109.0",
-    "@solana/web3.js": "1.36.0",
     "array-move": "^2.2.1",
     "bignumber.js": "^9.0.2",
     "bluebird": "^3.5.1",
+    "bs58": "^5.0.0",
```

Comment:
this can't be done in cpp somehow to avoid adding a new dep?

Reply:
That should be possible, but we'll need to modify the core API to support it.  Right now the backend expects addresses to be passed from the frontend as strings with the correct encoding.

Thoughts @yrliou?

Reply:
Looking into options here, one challenge is that we allow the user to select which accounts they want to add before anything is sent to the core side.

Also curious @diracdeltas, is vendoring this dep a possibility? I'm assuming we are mostly concerned about npm package takeover

Reply:
@nvonpentz vendoring doesn't help much since i assume we aren't going to carefully audit all of the code (and we already should be pinning an exact version anyway)

Reply:
i'm ok with this if it's really needed since it's not a huge dep AFAICT, but if it can be done without adding a new dep that'd be ideal

Reply:
Talked offline, we decided to add an API for base58 encoding a list of addresses.  That's implemented and I removed the bs58 dependency.